DomainEntity/Candidate.metaed

```diff
@@ -36,7 +36,7 @@ Domain Entity Candidate
   bool HispanicLatinoEthnicity
       documentation "An indication that the individual traces his or her origin or descent to Mexico, Puerto Rico, Cuba, Central, and South America, and other Spanish cultures, regardless of race. The term, ""Spanish origin,"" can be used in addition to ""Hispanic or Latino."""
       is optional
-  descriptor EdFi.OldEthnicity
+  descriptor OldEthnicity
```

Comment:
Updating OldEthnicity from a Ed-Fi project reference to the descriptor added above